<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetBalanceProofDialog</class>
 <widget class="QDialog" name="GetBalanceProofDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Get proof of balance</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/images/cryptonote</normaloff>:/images/cryptonote</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="m_amountSpin">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>0.00000000</double>
     </property>
     <property name="maximum">
      <double>99999999.00000000</double>
     </property>
     <property name="value">
      <double>0.00000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Signature</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="m_messageEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTextEdit" name="m_signatureEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="m_proofHorizontalLayout">
     <item>
      <widget class="QPushButton" name="m_copyProofButton">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveProofButton">
       <property name="text">
        <string>Save to file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_proofCloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_proofCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>GetBalanceProofDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_messageEdit</sender>
   <signal>textChanged()</signal>
   <receiver>GetBalanceProofDialog</receiver>
   <slot>genProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_amountSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>GetBalanceProofDialog</receiver>
   <slot>genProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyProofButton</sender>
   <signal>clicked()</signal>
   <receiver>GetBalanceProofDialog</receiver>
   <slot>copyProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_saveProofButton</sender>
   <signal>clicked()</signal>
   <receiver>GetBalanceProofDialog</receiver>
   <slot>saveProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
